### DISCUSSION TITLE
Bring back status online/offline icons

### DISCUSSION THREAD

**Discussion by mquadrat** [tailwindlabs/heroicons#795] [Ideas]
In v2 there are no icons to express online / offline status. Would be great if those could be brought over from the v1 release.

**Comment by angelaki** [comment]
Been a while 😉 Any news on this? Migrating a project right now.

**Comment by ndri** [comment]
They're called `signal` and `signal-slash` in v2.

**Comment by angelaki** [comment]
Great, thank you!
